oat-sa/tao-item-runner-qti-fe
# [TAO-10497] R&O Processing - Refactor qti item loader

Related to : https://oat-sa.atlassian.net/browse/TAO-10497
 
Refactor QTI item loader to recognize response processing type by response rules

**NOTE**:
QTI item loader was refactored to ES6 syntax this changes done in [separated commit](https://github.com/oat-sa/tao-item-runner-qti-fe/pull/104/commits/d64bb90b657b5f811452fedea7dd023aea6e151f) . Actual changes made in this [commit](https://github.com/oat-sa/tao-item-runner-qti-fe/pull/104/commits/48bd449d5d936e993aedfe98749e85785f0369b9)

#### How to test
 
- prepare an instance of TAO with `xmlEdit` and `xmlEditRp ` installed
- prepare an item with interactions with different response processing templates
- try to open item for authoring. make sure that response processing templates properly recognized
- prepare an item with custom response processing
- try to open item from authoring. make sure that response processing templates properly recognized
- prepare a delivery
- execute delivery as test taker
- check that test items loaded correctly
  
#### Dependencies
   
Companion PR :
 - [ ] https://github.com/oat-sa/extension-tao-itemqti/pull/1527
